packages/fxa-auth-server/lib/senders/emails/scss/style.scss

```diff
@@ -0,0 +1,30 @@
+$large-font-size: 20px;
+$small-font-size: 12px;
+
+.header-text div {
+  font-family: sans-serif !important;
+  font-size: $large-font-size !important;
+  line-height: 28px !important;
+  font-weight: normal !important;
+  text-align: center !important;
+}
+td {
+  padding: 4px !important;
+}
+span {
+  color: #0a84ff;
+}
+.primary-text div {
+  font-family: sans-serif !important;
+  font-size: $small-font-size !important;
+  line-height: 20px !important;
+  font-weight: normal !important;
+}
+.fxa-logo td {
+  height: 60px !important;
+  width: 60px !important;
+}
+.sync-logo td {
+  height: 176px !important;
+  width: 240px !important;
```

Comment:
One more note on these... do we know if we _have_ to use `!important` on all of these lines? I'm sort of not sure when we need them because of MJML or because of email clients. Maybe it's safer to just slap an `!important` on whatever had one previously, but dang, that hurts my soul because I guarantee you we've just randomly stuck that on a lot of inline styles without knowing if they're needed.

Reply:
Yeah.. I'd love to avoid !important unless absolutely necessary. Might need to spend some time going through each one to test if it really is needed.